pwndbg/commands/elf.py

```diff
@@ -40,48 +35,53 @@ def elfheader():
         for start, end, name in sections:
             print('%#x - %#x ' % (start, end), name)
 
+
 @pwndbg.commands.Command
+@pwndbg.commands.OnlyWithFile
 def gotplt():
     """
     Prints any symbols found in the .got.plt section if it exists.
     """
     print_symbols_in_section('.got.plt', '@got.plt')
 
+
 @pwndbg.commands.Command
+@pwndbg.commands.OnlyWithFile
 def plt():
     """
     Prints any symbols found in the .plt section if it exists.
     """
     print_symbols_in_section('.plt', '@plt')
 
+
 def get_section_bounds(section_name):
     local_path = pwndbg.file.get_file(pwndbg.proc.exe)
-    if not local_path:
-        print('No file is selected')
-        return (None, None)
 
     with open(local_path, 'rb') as f:
         elffile = ELFFile(f)
 
         section = elffile.get_section_by_name(section_name)
 
         if not section:
-            return (None, None)
+            return None, None
```

Comment:
Is there a PEP which specifies that return tuples should not be enclosed in parens?